src/functionality/search.ts

```diff
@@ -80,7 +80,10 @@ export function createSearch(view: MapView | SceneView, portal: Portal, searchCo
       }
     });
   } else {
-    searchConfiguration[INCLUDE_DEFAULT_SOURCES] = true;
```

Comment:
i think we need to keep this. This is for existing apps that have this set to `false` in the app data

Reply:
What do you mean? It's still there

<img width="229" alt="image" src="https://user-images.githubusercontent.com/45497237/219125483-8258be31-9d54-4a38-849b-23308c4a13d9.png">

Reply:
It's failing if it's null

<img width="474" alt="image" src="https://user-images.githubusercontent.com/45497237/219125746-6d75b1a3-453d-4c2f-b330-9996c0f618e1.png">